bin/git-archive-file

```diff
@@ -20,6 +20,8 @@ else
 	if [ "$BRANCH" = "master" ]; then
 		FILENAME=$ARCHIVE_NAME.$VERSION.zip
 	else
+	  # govern against invalid path characters
```

Comment:
Please follow the indentation of the other code.

Reply:
The indentation is exactly the same in my IDE, IntelliJ.
![19 12 23 09 04 51-git-extras  C__Repos_git-extras  -  _bin_git-archive-file - IntelliJ IDEA (Adm](https://user-images.githubusercontent.com/5083576/71365279-1ce5c900-2564-11ea-81c7-3b2d1f8b22bd.png)

Reply:
@DarekDan
Look like your IDE uses 2 spaces to represent a tab, and converts a tab into two space. Would you change the tabs around into 4 spaces?